src/components/ranking/ranking.js

```diff
@@ -0,0 +1,72 @@
+import './ranking.css';
+
+const sortedArr = [
+  { player: 'Anna', answers: 20, questions: 30 },
+  { player: 'Tomek', answers: 10, questions: 20 },
+  { player: 'Zenon', answers: 3, questions: 20 }
+];
+
+const getRankingTable = (sortedArr) => {
```

Comment:
this data `sortedArr` will be taken from local storage in the future?

Reply:
btw I think it should be renamed to something like: `sortedArr` => `playersRankingData` 
`getRankingTable` => `preparePlayersRankingTable`